.travis.yml

```diff
@@ -60,6 +60,7 @@ before_install:
 
 install:
   - pip install -r requirements.txt
+  - if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then pip install black; fi
```

Comment:
This should be handled in the `setup.py`'s `test_requires` instead...

Reply:
I agree that is a solution for the future, but we don't have a `setup.py` in `libres` though.

Reply:
Fixed in https://github.com/equinor/libres/pull/784 which introduces `test_requirements.txt`